src/pages/users/users-details/containers/user-details-body/user-details-body.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
It is a useless component because it just wraps one component in another without any logic